tools/run_tests/xds_k8s_test_driver/framework/infrastructure/traffic_director.py

```diff
@@ -40,6 +40,9 @@
 _NetworkServicesV1Alpha1 = gcp.network_services.NetworkServicesV1Alpha1
 EndpointConfigSelector = _NetworkServicesV1Alpha1.EndpointConfigSelector
 
+# Testing metadata consts
+_TEST_METADATA_KEY = 'xds_md'
```

Comment:
nit: I think we can be more straightforward about the variable name. Like call it `_TEST_AFFINITY_METADATA_KEY`.